src/main/java/com/shanebeestudios/skbee/elements/recipe/effects/EffBrewingRecipe.java

```diff
@@ -0,0 +1,93 @@
+package com.shanebeestudios.skbee.elements.recipe.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import com.shanebeestudios.skbee.api.recipe.RecipeUtil;
+import io.papermc.paper.potion.PotionMix;
+import org.bukkit.Bukkit;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.RecipeChoice;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Recipe - Brewing")
+@Description({"Registers a new custom brewing stand recipe",
+        "Input refers to the bottom 3 slots of a brewing stand",
+        "Note: brewing recipes are fairly new within paper, as such support for this is minimal.",
+        "there is currently no way to retrieve all potion recipes or get recipe information like ingredients",
+        "Requires PaperMC 1.18+"})
```

Comment:
```suggestion
@Description({"Registers a new custom brewing stand recipe.",
        "Input refers to the bottom 3 slots of a brewing stand.",
        "\nNote: brewing recipes are fairly new within Paper, as such support for this is minimal.",
        "There is currently no way to retrieve all potion recipes or get recipe information like ingredients.",
        "Requires PaperMC 1.18+"})
```
Just me being nit picky with formatting.